docs/ansible/roles/cryptsetup/defaults-detailed.rst

```diff
@@ -74,6 +74,18 @@ Each item of those lists is a dictionary with the following documented keys:
 
   Default to :envvar:`cryptsetup__use_uuid`.
 
+.. _cryptsetup__devices_check_with_checksum:
+
+``check_with_checksum``
+  Optional, boolean.
+  Compute checksum of the `ciphertext block device` given by
+  :ref:`item.ciphertext_block_device <cryptsetup__devices_ciphertext_block_device>`.
+
+  Setting this to ``False`` for (very large) regular files used as `ciphertext block device` 
+  avoids long checksum computation times.
+
+  Default to ``True``.
```

Comment:
Do we actually need to make this configurable? Is the checksum used anywhere?

Reply:
No, the checksum computed by the `stat` module is not used elsewhere in the `cryptsetup` role.

It would surely be possible to disable the checksum by default with `get_checksum: 'False'`. If that is ok I will update the PR accordingly. Please advise.

Reply:
Yes, that is what I meant. Checksumming can be disabled. Thank you for bringing that up! I just forgot to set the best options for this module and never noticed …